app/styles/_library/_objects.blocks.scss

```diff
@@ -69,10 +69,11 @@
 }
 
 .c-block__dek p {
-  line-height: 1.45;
+  line-height: var(--line-height-4);
 
   @include media("<=medium") {
-    @include typeface(body, 5)
+    @include typeface(body, 5);
+    line-height: var(--line-height-4);
```

Comment:
It looks like this is *almost* the only place we're using font size 5. It would be nice if we could just change the value of --line-height-5 to be what we want instead of having a mismatched line height / font size here. Not sure of the side effects (i think 5 is used one other place in the footer right now), but something to think about